modules/va_facilities/spec/requests/facilities_request_spec.rb

```diff
@@ -58,14 +58,16 @@ def query_params(url)
       expect(json['meta']['distances']).to eq([])
     end
 
-    it 'responds to GET #index with lat/long' do
+    it 'responds to GET #index with lat/long sorted by distance' do
```

Comment:
Would it make sense to add "ascending" to the end here so that future developers understand what we intended from description alone?